src/index.js

```diff
@@ -1,11 +1 @@
 import './style.css';
```

Comment:
- [ ] At this point of the project you should have developed a working version of the application as the project requirements said. Kindly take a look at this screenshot and to project requirements. 😸 
    - All the buttons should be functional, which means, the submit button should add a score to the list, and the refresh button should refresh the list of scores 🚀
    - Remember to use ES6 syntax
    ![imagen](https://user-images.githubusercontent.com/60789317/136084215-0b0d88af-7940-43a2-84e5-4ab5548e93cf.png)

Reply:
Hi, thank you for the feedback.
this is the first milestone for the entire project and it says we should just add basic `HTML `markup and little styling. It doesn't state we should add functionality to the button yet.

Kindly find the link to the project requirement for the first milestone 
https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md